package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "website",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "prettier": "npx prettier --write ."
+  },
+  "engineStrict": true,
+  "engines": {
+    "node": ">=14.0.0",
+    "npm": ">=8.0.0",
+    "yarn": "please-use-only-npm"
+  },
+  "dependencies": {
+    "@next/font": "13.1.6",
+    "@types/node": "18.11.18",
+    "@types/react": "18.0.27",
+    "@types/react-dom": "18.0.10",
+    "eslint": "8.33.0",
+    "eslint-config-next": "13.1.6",
+    "next": "13.1.6",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "typescript": "4.9.4"
+  },
+  "devDependencies": {
+    "eslint-config-prettier": "^8.6.0",
+    "lint-staged": "^13.1.0",
+    "prettier": "2.8.3"
```

Comment:
Since you have prettier already, please also add the prettierrc file as config for prettier. Could you take a look at our last repos?

Reply:
Adding .prettierrc & .prettierignore files.